src/myrtlespeech/builders/task_config.py

```diff
@@ -91,6 +91,48 @@ def build(
 
     seq_to_seq.optim = optim
 
+    # create learning rate scheduler
```

Comment:
Suggest moving this to a separate function outside of `build` that returns a scheduler, e.g. `def _create_lr_scheduler(...)` that means this can be refactored to:

```python
seq_to_seq.lr_scheduler = _create_lr_scheduler(...)
```

Yes the function currently builds the optimizer, data loader etc but they should have ideally been written in the above style too to separate out the concerns.